third_party/2and3/boto/kms/layer1.pyi

```diff
@@ -0,0 +1,42 @@
+# Stubs for boto.kms.layer1 (Python 3.6)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any, Dict, List, Optional, Type
+from boto.connection import AWSQueryConnection
+
+class KMSConnection(AWSQueryConnection):
+    APIVersion = ...  # type: str
+    DefaultRegionName = ...  # type: str
+    DefaultRegionEndpoint = ...  # type: str
+    ServiceName = ...  # type: str
+    TargetPrefix = ...  # type: str
+    ResponseError = ...  # type: Type[Exception]
+    region = ...  # type: Any
+    def __init__(self, **kwargs) -> None: ...
+    def create_alias(self, alias_name: str, target_key_id: str): ...
+    def create_grant(self, key_id: str, grantee_principal: str, retiring_principal: Optional[str] = ..., operations: Optional[List] = ..., constraints: Optional[Dict] = ..., grant_tokens: Optional[List] = ...): ...
+    def create_key(self, policy: Optional[str] = ..., description: Optional[str] = ..., key_usage: Optional[str] = ...): ...
+    def decrypt(self, ciphertext_blob: bytes, encryption_context: Optional[Dict] = ..., grant_tokens: Optional[List] = ...): ...
+    def delete_alias(self, alias_name: str): ...
+    def describe_key(self, key_id: str): ...
+    def disable_key(self, key_id: str): ...
+    def disable_key_rotation(self, key_id: str): ...
+    def enable_key(self, key_id: str): ...
+    def enable_key_rotation(self, key_id: str): ...
+    def encrypt(self, key_id: str, plaintext: bytes, encryption_context: Optional[Dict] = ..., grant_tokens: Optional[List] = ...): ...
+    def generate_data_key(self, key_id: str, encryption_context: Optional[Dict] = ..., number_of_bytes: Optional[int] = ..., key_spec: Optional[str] = ..., grant_tokens: Optional[List] = ...): ...
```

Comment:
The documentation says "map" as the type of the `encryption_context` argument, which I assume means it accepts any mapping. Maybe we should type it as `Mapping[str, Any]`? (Not sure what exactly the values are supposed to be.) In general it's also good to add type arguments for precision instead of just "Dict" or "List".